### PR TITLE
Added message feature (Fixes #7)

### LINKED ISSUE #7: Sending a message to a member
Implementing as a method on the OpenHumansMember model the ability to send a message to a member via API. (This might be used to notify a member that an analysis has been completed and a report is available, or that files are available, etc.)

### PR DESCRIPTION
**Pull request by DartingMelody**
Added message feature .

### CODE REVIEW

**Comment by DartingMelody** [comment]
Hi @madprime , please review the PR. Thanks :)

File: openhumans/models.py [hunk 1]
```diff
@@ -98,3 +98,13 @@ def _refresh_tokens(self, client_id, client_secret):
             self.refresh_token = data['refresh_token']
             self.token_expires = self.get_expiration(data['expires_in'])
             self.save()
+
+    def message(self, subject, message, access_token, all_members=False,
```
Comment (madprime) [inline openhumans/models.py]: It doesn't make sense to include all_members or project_member_ids here, the message feature in this context is only for the user represented by the model, and not other users.

**Comment by DartingMelody** [comment]
Hi @madprime , I have limited message feature to models as discussed. Please have a look. Thanks :)

**Comment by madprime** [comment]
(and thank you @DartingMelody!)

**Pull request merged**